packages/tools/pactjs-cli/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log - @kadena/pactjs-cli
 
+## 1.3.0
+
+### Patch Changes
+
+- e8d0e92a: add missing shebang to the executable file
```

Comment:
Rather word this as `fix issue where shebang was missing in an executable`

See also: https://rushjs.io/pages/best_practices/change_logs/#recommended-practices:~:text=Fix%20an%20issue%20with/where